src/locales/fr/PAGES/SCOPE.json

```diff
@@ -0,0 +1,28 @@
+{
+    "TITLE": "Étape 1 : Déterminer le périmètre de l’évaluation",
+    "INTRO": "Définir les paramètres généraux et le périmètre de l’évaluation. L’idéal est de le faire avec la personne qui a commandé l’évaluation (qui peut ou non être le propriétaire du site Web), afin de garantir des attentes communes quant au périmètre de l’évaluation.",
+    "RESOURCE_LINK_NAME": "Étape 1 : Déterminer le périmètre de l’évaluation",
+    "LABEL_SITE_NAME": "Nom du site Web",
+    "INF_SITE_NAME": "Proposez un nom pour le site Web que vous souhaitez évaluer. Par exemple : <br> « Site Web public de l’organisation Exemple » <br> « Boutique en ligne de l’entreprise Exemple » <br> « Intranet de l’université Exemple »",
+    "LABEL_SITE_SCOPE": "Portée du site Web",
+    "INF_SITE_SCOPE_0": "Définir le périmètre du site Web, de manière à ce que les pages Web incluses dans l’évaluation soient claires. Pour plus d’informations, voir [WCAG-EM Étape 1.a : Définir le champ d’application du site Web](https://www.w3.org/TR/WCAG-EM/#step1a). Par exemple :",
+    "INF_SITE_SCOPE_LI0": "« Tout le contenu du site Web public de l’organisation d’exemple, situé à l’adresse `http://www.example.org` »",
+    "INF_SITE_SCOPE_LI1": "« Tout le contenu Web de la boutique en ligne de l’organisation d’exemple, située à `http://www.example.org/shop/` »",
+    "INF_SITE_SCOPE_LI2": "« Tout le contenu Web de la version mobile du site Web public de l’organisation d’exemple. situé à `http://m.example.org` »",
+    "INF_SITE_SCOPE_1": "WCAG-EM Étape 1.a : Définir le périmètre du site Web <i class='fas fa-external-link-alt'></i>",
+    "LABEL_WCAG_VERSION": "Version du WCAG",
+    "INFO_WCAG_VERSION": "Sélectionner la version du WCAG à utiliser. Version 2.1 (par défaut) ou 2.0",
+    "WCAG21": "WCAG 2.1",
+    "WCAG20": "WCAG 2.0",
+    "LABEL_CONFORMANCE_TGT": "Niveau de conformité",
+    "INF_CONF_TGT": "Sélectionnez un niveau de conformité WCAG 2 cible (A, AA ou AAA) pour l’évaluation. Pour plus d’informations, voir [WCAG-EM Étape 1.b : Définir la cible de conformité](https://www.w3.org/TR/WCAG-EM/#step1b). Cette sélection détermine les filtres de niveau de conformité qui sont actifs par défaut à l’ « Étape 4 : Auditer l’échantillon ».",
+    "LABEL_SUPPORT_BASE": "Environnement de réference pour évaluer l'accessibilité",
+    "INF_SUPPORT_BASE": "Définir les navigateurs Web, les technologies d’assistance et les autres [agents utilisateurs](https://www.w3.org/Translations/WCAG20-fr/#useragentdef) qui seront [compatibles avec l’accessibilité](https://www.w3.org/Translations/WCAG20-fr/#accessibility-supporteddef) par le site Web. Par exemple, « Internet Explorer (IE) avec JAWS », « FireFox avec NVDA » et « Apple avec VoiceOver » pourraient être des définitions de base. Pour plus d’informations, voir [WCAG-EM Étape 1.c : Définir une référence de support d'accessibilité](https://www.w3.org/TR/WCAG-EM/#step1c).",
```

Comment:
```suggestion
    "INF_SUPPORT_BASE": "Définir les navigateurs Web, les technologies d’assistance et les autres [agents utilisateurs](https://www.w3.org/Translations/WCAG20-fr/#useragentdef) qui seront [compatibles avec l’accessibilité](https://www.w3.org/Translations/WCAG20-fr/#accessibility-supporteddef) par le site Web. Par exemple, « Internet Explorer (IE) avec JAWS », « FireFox avec NVDA » et « Apple avec VoiceOver » pourraient être des définitions de base. Pour plus d’informations, voir [WCAG-EM Étape 1.c : Définir une référence pour la prise en charge de l’accessibilité](https://www.w3.org/TR/WCAG-EM/#step1c).",
```

Reply:
This suggestion is marked "Outdated" by GitHub and has not been applied. @KodGit Can you commit this version: https://github.com/w3c/wai-wcag-em-report-tool/pull/127#discussion_r1414159651?